src/main/java/org/jenkinsci/plugins/workflow/actions/ErrorAction.java

```diff
@@ -140,7 +145,7 @@ public String getUrlName() {
      */
     public static @CheckForNull FlowNode findOrigin(@NonNull Throwable error, @NonNull FlowExecution execution) {
         FlowNode candidate = null;
-        for (FlowNode n : new ForkScanner().allNodes(execution)) {
+        for (FlowNode n : new DepthFirstScanner().allNodes(execution)) {
```

Comment:
Unrelated, but there is no reason to use `ForkScanner` here and it has various potential issues if used on an incomplete build.

Reply:
Actually this is not ok, but I guess we have no tests for the relevant case. We need to make sure that we only replace the candidate if the new node is a parent of the candidate node. I guess I'll just revert this for now.

Reply:
https://github.com/jenkinsci/workflow-api-plugin/pull/327/commits/28d0acd535d3008120412de33819115953edf710